lib/ansible/modules/source_control/git.py

```diff
@@ -169,6 +169,15 @@
               can be separated from working tree.
         version_added: "2.7"
 
+    valid_pgpkeys:
```

Comment:
Maybe
```suggestion
    valid_pgp_keys:
```
Or even
```suggestion
    valid_pgp_fingerprints:
```
?

I think this naming could be improved a bit.

Reply:
Also, I don't want to use PGP term since it's confusing: Git itself uses GPG in its commands.

Reply:
Maybe something with whitelist/trusted.
```suggestion
    trusted_gpg_keys:
```
```suggestion
    trusted_gpg_fingerprints:
```
```suggestion
    gpg_whitelist:
```

Reply:
gpg_whitelist seems logical!

Reply:
Thanks!